SpatialGDK/Source/SpatialGDK/Public/Schema/Interest.h

```diff
@@ -6,8 +6,8 @@
 
 namespace SpatialGDK
 {
-
 using EdgeLength = Coordinates;
+using FrequencyToConstraintsMap = TMap<float, TArray<struct QueryConstraint>>;
```

Comment:
What is going on here

Reply:
this is a `using` for the new type
It can't be in common types because at that point we don't know what a query constraint is
so we put it in here, where we define other interest related types
`QueryConstraint` is forward declared because it is lower down in the file

Reply:
for posterity: it totally could be in common types, I just had/have no idea how forward declarations work. it was moved to common types